tests/ops/op_math/test_controlled_ops.py

```diff
@@ -49,9 +49,61 @@
 X_broadcasted = np.array([X] * 3)
 
 
+# pylint: disable=too-many-public-methods
 class TestControlledQubitUnitary:
     """Tests specific to the ControlledQubitUnitary operation"""
 
+    def test_wires_is_none(self):
+        base_op = [[0, 1], [1, 0]]
+
+        with pytest.raises(TypeError, match="Must specify a set of wires"):
+            qml.ControlledQubitUnitary(base_op, control_wires=1, wires=None)
+
+    @pytest.mark.jax
+    @pytest.mark.usefixtures("enable_disable_plxpr")
+    def test_wires_specified_twice_with_capture(self):
+        base = qml.QubitUnitary(X, wires=0)
+        with pytest.warns(
+            UserWarning,
+            match="base operator already has wires; values specified through wires kwarg will be ignored.",
+        ):
+            qml.ControlledQubitUnitary(base, control_wires=[1, 2], wires=3)
+
+    @pytest.mark.jax
+    @pytest.mark.usefixtures("enable_disable_plxpr")
+    @pytest.mark.parametrize(
+        "control_wires, wires",
+        [(0, 1), ([0, 1], [2])],
+    )
+    def test_consistency_with_capture(self, control_wires, wires):
+        base_op = [[0, 1], [1, 0]]
+
+        op = qml.ControlledQubitUnitary(base_op, control_wires=control_wires, wires=wires)
+        assert op.base.wires == Wires(wires)
+        assert op.control_wires == Wires(control_wires)
+
+    @pytest.mark.jax
+    @pytest.mark.usefixtures("enable_disable_plxpr")
+    @pytest.mark.parametrize(
+        "control_wires_1, wires_1, control_wires_2, wires_2",
+        [
+            ([0, 1], [2], [0, 1, 2], ()),  # Pair of configurations to compare
+        ],
+    )
+    def test_pairwise_consistency_with_capture(
+        self, control_wires_1, wires_1, control_wires_2, wires_2
+    ):
+        base_op = [[0, 1], [1, 0]]
+
+        op_1 = qml.ControlledQubitUnitary(base_op, control_wires=control_wires_1, wires=wires_1)
+        op_2 = qml.ControlledQubitUnitary(base_op, control_wires=control_wires_2, wires=wires_2)
+
+        assert op_1.base.wires == Wires(2)
+        assert op_2.base.wires == Wires(2)
+
+        assert op_1.control_wires == Wires([0, 1])
+        assert op_2.control_wires == Wires([0, 1])
```

Comment:
If there's only one test case in the parametrization, then it's probably not worth using `pytest.mark.parametrize` and just define the variables inside the test.

Reply:
Additionally, if you've using values instead of variables in the assertions, it kind of defeats the purpose of parametrization.

Reply:
Addressed in latest commit!